README.md

```diff
@@ -19,6 +23,8 @@ Follow the instructions of the Blender Development plugin setup to connect to yo
 ## Installing in Blender
 If you simply want to test how the addon works, get the latest version of the addon from the [releases page](https://github.com/UncleFirefox/Measures-Addon-For-Blender/releases).
 
+**Remember, only Blender 2.90 works with this plugin**
+
 Inside Blender go to: `Edit > Preferences`, go the the `Add-ons` section in the left menu, hit the `Install...` button and select the file you donwloaded.
 
```

Comment:
At least for me when I tried to install it from the release zip file I needed to remove first the release number version from the zip and the folder inside since it was giving some error when importing related with the `.` in between. Maybe warn about this in the README

Reply:
Yeah, mea culpa, installs now will have a folder of type Measures-Addon-For-Blender-xxx-yyy-zzz which will not make the install blow up, that way multiple versions of the plugin can coexist and you can choose to enable/disable the plugin at will when you are ready to move on.